tripal/src/Plugin/TripalPubLibrary/TripalPubLibraryPubMed.php

```diff
@@ -27,6 +27,16 @@ class TripalPubLibraryPubMed extends TripalPubLibraryBase {
    */
   protected array $webquery = [];
 
+  /**
+   * Options for file retrieval from NCBI
+   *
+   * @var array
```

Comment:
We should indicate more information about what is in this array.

```suggestion
   * @var array
   *   Options to be passed to the file retrieval service.
   *   @see Drupal\tripal\Services\TripalFileRetriever::retrieveFileContents()
```